tools/chocolateyuninstall.ps1

```diff
@@ -1,5 +1,11 @@
-Uninstall-ChocolateyZipPackage 'rgsupervision' 'rgsupv-win.zip'
-
 If (Test-Path -Path "C:\Program Files (x86)\RG-Supervision\RG_Supervision.exe") {
-	& "C:\Program Files (x86)\RG-Supervision\RG_Supervision.exe" --action uninstall-console
+	$packageArgs = @{
+		packageName    = "rgsupervision"
+		fileType       = "exe"
+		file           = "C:\Program Files (x86)\RG-Supervision\RG_Supervision.exe"
+		silentArgs     = "--action uninstall-console"
+		validExitCodes = @(0, 3010, 1641)
```

Comment:
What are theses codes?

Reply:
Hmmm probably kept it from chocolatey doc but after quick check it looks like MSI packages error codes
Don't know RG_Supervision.exe possible exit code, should I keep 0 and 1 ?

Reply:
Well well, no need to know all possible codes, only 0 is success I guess so > removed
![image](https://user-images.githubusercontent.com/18307823/185346248-06489128-3c5d-4a99-863a-2348af207357.png)

Reply:
Here is the code describing it: https://github.com/rgsystemes/rgsupv-agent/blob/master/agent/main.cpp#L330-L338, which refers to CLIController, which only use EXIT_SUCCESS (0) or EXIT_FAILURE (1).